fastlane/Fastfile

```diff
@@ -281,3 +278,24 @@ before_all do |lane|
   end
   # rubocop:enable Style/IfUnlessModifier
 end
+
+def compute_release_branch_name(options:, version: release_version_current)
+  branch_option = :branch
+  branch_name = options[branch_option]
+
+  if branch_name.nil?
+    branch_name = release_branch_name(version:)
+    UI.message("No branch given via option '#{branch_option}'. Defaulting to #{branch_name}.")
+  end
+
+  branch_name
+end
```

Comment:
I moved this in `Fastfile` because it will be shared between lanes across files. 

I don't remember why we need this computation logic over using the simpler interpolation from `release_branch_name(version:)`.

I decided to roll with it instead of investigating to make faster progress with the release automation.